app-emulation/wineasio/wineasio-9999.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 1999-2010 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI="5"
+[[ "${PV}" = "9999" ]] && inherit subversion
```

Comment:
You may set up a variable e.g. SVN=subversion; and append '$SVN' to a single 'inherit' line; unset the variable afterward; and then remove 'src_unpack()' altogether.

FEWER LINES = LESS MAINTENANCE